### ISSUE TITLE
fix: reduce the size bundle of the library

### ISSUE THREAD

**Issue by TahimiLeonBravo** [nexxtway/react-rainbow#1339]
for performance problems, we should reduce the size of the library when it is imported

**Comment by ldelia** [comment]
Any progress on this? webpack-bundle-analyzer is reporting that the size of the react-rainbow-components library is 3.3 MB in bundle.js

**Comment by VincentVanBork** [comment]
Is it really and Issue ? I know it might be problematic to import with path like 'react-rainbow-components/components/<component_name>/index.js but this way you reduce size significantly. And bundle drops like from 3 MiB to few hundred KiB

**Comment by busybox11** [comment]
Any updates on this?
When I import only `Calendar` and `RainbowThemeContainer`, my webpack bundle has the entire library in it and a huge unused `xlsx` dependency.
Is it possible to prevent this?

**Comment by LeandroTorresSicilia** [comment]
@busybox11 Right now the only way you can solve this is like @VincentVanBork mentioned here: https://github.com/nexxtway/react-rainbow/issues/1339#issuecomment-843418137
You can import it like this:
```import Calendar from 'react-rainbow-components/components/Calendar';
this will cause you only import the Calendar instead of all the library.

**Comment by LeandroTorresSicilia** [comment]
We just released a new version 1.31.0 which fixes this issue, now you can get a smaller bundle size using webpack or create react app out of the box only with the things you imported and not all the library as before.

Thanks for using react-rainbow
